mqtt-pushgateway.py

```diff
@@ -62,7 +63,7 @@ def update(self, topic, value):
             self.value = float(value)
             self.is_numeric = True
         except (TypeError, ValueError):
-            self.value = value.decode("utf-8")
+            self.value = value
```

Comment:
not decoding the payload breaks string metrics (`state="CLOSED"` becomes `state="b'CLOSED'"` in the output). However, you're right in the sense that this isn't the optimal place to do it. I refactored that part of the code, please update your PR to master. Sorry for the fuss :(